crates/anvil/src/config.rs

```diff
@@ -694,10 +695,10 @@ impl NodeConfig {
         self
     }
 
-    /// Sets the `fork_block_number` to use to fork off from
+    /// Sets the `fork_choice` to use to fork off from
     #[must_use]
-    pub fn with_fork_block_number<U: Into<u64>>(mut self, fork_block_number: Option<U>) -> Self {
```

Comment:
I'd like to keep this function but this would then just call the new `with_fork_choice` function